<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCapture">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Captured Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>311</width>
      <height>229</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>Video Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoFrame_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>311</width>
      <height>229</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>Video Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrint">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string extracomment="Will add later On"/>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TextFieldAddText">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddText">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string extracomment="Will add later On"/>
    </property>
    <property name="text">
     <string>Add Text</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
